build.sbt

```diff
@@ -188,4 +189,16 @@ lazy val java = project
     name := "otel4s-java"
   )
 
+lazy val examples = project
+  .enablePlugins(NoPublishPlugin)
+  .in(file("examples"))
+  .dependsOn(core.jvm, java)
+  .settings(
+    name := "otel4s-examples",
+    libraryDependencies ++= Seq(
+      "io.opentelemetry" % "opentelemetry-sdk" % OpenTelemetryVersion,
+      "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % s"${OpenTelemetryVersion}-alpha"
```

Comment:
I wonder what Scala Steward will do with this.  Probably doesn't matter, as long as OpenTelemetryVersion gets bumped.